src/relay/backend/graph_executor_codegen.cc

```diff
@@ -270,8 +274,13 @@ class GraphExecutorCodegen : public backend::MemoizedExprTranslator<std::vector<
           std::make_pair(static_cast<int>(param_storage_ids_[param.first]), param.second)));
     }
     ret.function_metadata = std::move(function_metadata_);
-    ret.lowered_funcs = lowered_module.per_target_module;
-    ret.external_mods = lowered_module.external_mods;
+
+    Optional<Array<tvm::runtime::Module>> external_modules =
+        lowered_mod->GetAttr<Array<tvm::runtime::Module>>("external_mods");
+    // This is the point where we separate the functions in the module by target
```

Comment:
You're right, I'll add it in here too. Since I'm going to be changing this part of the code a bunch I am going to leave the checks in for my own sanity :)